website/content/docs/release-notes/v1_11.mdx

```diff
@@ -0,0 +1,37 @@
+---
+page_title: Packer v1.11 release notes
+description: Learn about the changes in Packer v 1.11
+---
+
+# Packer v1.11 release notes
+
+This page describes changes to Packer in v1.11. Refer to the [Packer repository](https://github.com/hashicorp/packer/releases) for information about all releases.
+
+## New features
+
+This release includes the following updates.
+
+### Predictable plugin loading
+
+To boost stability, Packer no longer loads plugin binaries installed outside of its plugin directory. Additionally, Packer now only loads binaries that have an accompanying SHA256SUM file. In previous versions, Packer automatically loads plugins placed in one of the known directories when the file is named according to one of the following naming conventions:
```

Comment:
As below, I would suggest adopting the same convention without mentioning stability.

```suggestion
Packer no longer loads plugin binaries installed outside of its plugin directory. Additionally, Packer now only loads binaries that have an accompanying SHA256SUM file. In previous versions, Packer automatically loads plugins placed in one of the known directories when the file is named according to one of the following naming conventions:
```